srtcore/buffer_rcv.cpp

```diff
@@ -237,7 +237,7 @@ int CRcvBuffer::dropUpTo(int32_t seqno)
     // Move forward if there are "read/drop" entries.
     releaseNextFillerEntries();
     // Set nonread position to the starting position before updating,
-    // because start position was increased, and preceeding packets are invalid. 
+    // because start position was increased, and proceeding packets are invalid.
```

Comment:
No, this is wrong. "Preceding" was meant here.

Reply:
Concur with @ethouris, but it is spelled "preceding" and refers to packets BEFORE the new start position.

Reply:
I see, thanks for the explanation.